solutions/ha-nva/layer-7-ingress/azuredeploy.json

```diff
@@ -0,0 +1,801 @@
+            "maxLength": 64,
+            "metadata": {
+                "description": "Passphrase to access SSH Key (New key will be stored in Key Vault)"
+            }
+        },
+        "NvaNetworkSecurityGroupName": {
+            "defaultValue": "nsg-NVA",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Network Security Group for the NVA Resources"
+            }
+        },
+        "NvaVMNameBase": {
+            "defaultValue": "vm-nva",
+            "type": "String",
+            "metadata": {
+                "description": "Prefix for naming NVA VMs"
+            }
+        },
+        "NvaVMCount": {
+            "defaultValue": 2,
+            "type": "Int",
+            "allowedValues": [
+                1,
+                2,
+                3,
+                4,
+                5
+            ],
+            "metadata": {
+                "description": "How many NVA VMs to provision"
+            }
+        },
+        "NvaVMSize": {
+            "defaultValue": "Standard_D2s_v3",
+            "type": "String",
+            "allowedValues": [
+                "Standard_D2s_v3",
+                "Standard_D4s_v3"
+            ],
+            "metadata": {
+                "description": "VM Size for NVA VMs"
+            }
+        },
+        "WebNetworkSecurityGroupName": {
+            "defaultValue": "nsg-Web",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Network Security Group for the Web Resources"
+            }
+        },
+        "WebVMNameBase": {
+            "defaultValue": "vm-web",
+            "type": "String",
+            "metadata": {
+                "description": "Prefix for naming Web VMs"
+            }
+        },
+        "WebVMCount": {
+            "defaultValue": 2,
+            "type": "Int",
+            "minValue": 0,
+            "maxValue": 20,
+            "metadata": {
+                "description": "How many Web VMs to provision"
+            }
+        },
+        "WebVMSize": {
+            "defaultValue": "Standard_D2s_v3",
+            "type": "String",
+            "allowedValues": [
+                "Standard_D2s_v3",
+                "Standard_D4s_v3"
+            ],
+            "metadata": {
+                "description": "VM Size for NVA VMs"
+            }
+        },
+        "DmzNetworkSecurityGroupName": {
+            "defaultValue": "nsg-DMZ",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Network Security Group for the Dmz Resources"
+            }
+        },
+        "AllowDmzConnectionFromIPOrCIDRBlock": {
+            "defaultValue": "0.0.0.0/0",
+            "type": "string",
+            "metadata": {
+                "description": "Default value of 0.0.0.0/0 allows management and connections from the entire Internet"
+            }
+        },
+        "DeployAzureBastionDmz": {
```

Comment:
Consider updating too bool which will simplify the condition (applies here and elsewhere).